decidim-meetings/app/services/decidim/meetings/calendar/organization_calendar.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module Calendar
+      # This class handles how to convert an organization meetings to the
+      # ICalendar format. It finds the public meeting components for that
+      # organization and delegates the work to the `ComponentCalendar` class.
+      class OrganizationCalendar < BaseCalendar
+        # Renders the meetings in an ICalendar format. It caches the results in
+        # Rails' cache.
+        #
+        # Returns a String.
+        def events
+          @events ||= components.map do |component|
+            ComponentCalendar.new(component).events
+          end.compact.join
```

Comment:
Maybe use `flat_map` instead of join?

Reply:
I need this to be a single string, `flat_map` will give me an Array